data-prepper-plugins/otel-trace-group-processor/README.md

```diff
@@ -0,0 +1,67 @@
+# OTel Trace Group Processor
+
+This is a processor that fills in the missing trace group related fields in the collection of [Span](../../data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/Span.java) records output by [otel-trace-raw-processor](../otel-trace-raw-processor) and then convert them back into a new collection of string records.
```

Comment:
`... and then convert them back into a new collection of string records.` <- This is no longer accurate and we should be able to remove it.

Reply:
Great catch!